dspace/config/registries/project-types.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  This document was based on the OpenAIRE4 Guidelines for Literature Repositories
+  https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/index.html
+  -->
+  
+<dspace-dc-types>
+    <dspace-header>
+        <title>Project fields definition for OpenAIRE 4</title>
```

Comment:
Same here, it's slightly odd to be backporting a Project schema which is used for Configurable Entities (Project entity).

Reply:
We could choice other metadata schema to fill this information. In my opinion this is a data migration stuff, not to discuss here.

Reply:
My main question here would be "is this project information *required* for OpenAIREv4 compatibility"?  If so, then maybe that's a clearer argument to why we need a schema for this. However, if it is *not* required, it makes me wonder if we should just leave it out & advertise that v7 will provide Project-related entities & metadata.

Reply:
well, it is MA mandatory if applicable but for openaire the project reference is almost the most important piece of information. Institution already collect such details in the most different way, I appreciate here the attempt to suggest them to don't reinvent the wheel and use the metadata that we have already identified for version 7. 
In any case, we will need to say something to institution about how to migrate their extra information also for DSpace 7 because it is clear that any information that institution all around the world have collected about publication-project links should be "migrated" to work at the best.

Again for this PR, I feel ok with keeping the inputform changes only suggestion (commented out) and documenting them. Including these schema would make simpler for institution to turn them on but if this is controversial we can always reference them to the github master branch

Reply:
I still think this is odd, but I won't vote -1 here.  As long as this is *disabled by default* and there's clear documentation on how to use it if you need it.  I still feel though that it is not a best practice to backport a metadata schema that is **not** fully finalized for 7.0 (as 7.0 has not yet been released).  In other words, if the 7.0 schema changes these fields even slightly, this will invalidate this 6.x backport.